src/js/huron.js

```diff
@@ -34,22 +33,82 @@ class InsertNodes {
   }
 
   /**
+   * Apply a modifier to a render function
+   *
+   * @param {string} modifier - target modifier
+   * @param {object} meta - module metadata
+   */
+  applyModifier(modifier, meta) {
+    let rendered = false;
+    let data = meta.data;
+
+    if (data) {
+      // If we have a modifier, use it, otherwise use the entire data set
```

Comment:
It's implicit, but yes (based on whether or not you have a `data-huron-modifier` attribute)